packages/components/modules/messages/ChatRoom/ChatRoomHeader/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC } from 'react'
+
+import {
+  AvatarWithPlaceholder,
+  IconButton,
+  Iconify,
+  useResponsive,
+} from '@baseapp-frontend/design-system'
+
+import { Box, Typography } from '@mui/material'
+import { useFragment } from 'react-relay'
+
+import { ChatRoomHeaderFragment$data } from '../../../../__generated__/ChatRoomHeaderFragment.graphql'
+import { useChatRoom } from '../../context'
+import { ChatRoomHeaderFragment } from '../../graphql/queries/ChatRoomHeaderFragment'
+import useNameAndAvatar from '../../hooks/useNameAndAvatar'
+import { getParticipantCount, isGroupChat } from '../../hooks/utils'
+import { ChatHeaderContainer } from './styled'
+import { ChatRoomHeaderProps } from './types'
+
+const getSubtitle = (roomHeader: ChatRoomHeaderFragment$data) => {
```

Comment:
we normally place this util function under `ChatRoomHeader/utils.ts`

Reply:
@anicioalexandre Then the linter complains that `getSubtitle` is used before it is defined. Should I still move it below and suppress this error or rather keep it where it is?